extension/algo/src/function/strongly_connected_components.cpp

```diff
@@ -254,6 +258,7 @@ static offset_t tableFunc(const TableFuncInput& input, TableFuncOutput&) {
             initBwdColoringVertexCompute);
         GDSUtils::runAlgorithmEdgeCompute(input.context, bwdComputeState, graph,
             ExtendDirection::BWD, config.maxIterations);
+        progressBar->updateProgress(input.context->queryID, progress);
```

Comment:
I wonder if we need to be fine-grained as updating progress for each direction. Would update progress for each iteration be good enough? For that, we can update the progress at the end of this function, as there are still some VertexCompute after this line.